sycl/doc/extensions/proposed/sycl_ext_intel_cache_controls.asciidoc

```diff
@@ -309,8 +282,6 @@ is undefined.
 
 == Implementation notes
 
-It is intended that the SYCL cache control properties will be used by the
-compiler to generate SPIR-V cache control operations. Alternatively, the
-properties could be implemented by generating intrinsic function calls
-that match the cache control types.
+The SYCL cache control properties will be used by the
+compiler to generate SPIR-V cache control operations.
```

Comment:
I'm not sure how useful this implementation note is now.  Maybe just delete?

Reply:
Ok, deleted.